tests/pytorch/test_sanity.py

```diff
@@ -83,6 +98,12 @@ def custom_amax_compute(amax_history: torch.Tensor) -> torch.Tensor:
     return torch.min(amax_history, dim=0).values
 
 
+def reset_rng_states() -> None:
+    """revert back to initial RNG state."""
+    torch.set_rng_state(_cpu_rng_state)
+    torch.cuda.set_rng_state(_cuda_rng_state)
```

Comment:
`_cpu_rng_state` and `_cuda_rng_state` aren't set. Specifying `global` would also help make the intent clear:
```suggestion
    global _cpu_rng_state, _cuda_rng_state
    torch.set_rng_state(_cpu_rng_state)
    torch.cuda.set_rng_state(_cuda_rng_state)
```